test/auro-pane.test.js

```diff
@@ -3,26 +3,96 @@ import sinon from 'sinon';
 import '../src/auro-pane.js';
 
 describe('auro-pane', () => {
-  it('sets the CSS class on auro-pane > div element', async () => {
+  it('is accessible', async () => {
     const el = await fixture(html`
-      <auro-pane cssclass="testClass"></auro-pane>
-    `);
-
-    const div = el.shadowRoot.querySelector('div');
-    expect(div.className).to.equal('testClass');
-  });
-
-  it('auro-pane is accessible', async () => {
-    const el = await fixture(html`
-      <auro-pane cssclass="testClass"></auro-pane>
+      <auro-pane date="2020-09-09"></auro-pane>
     `);
 
     await expect(el).to.be.accessible();
   });
 
-  it('auro-pane custom element is defined', async () => {
+  it('defines the custom element', async () => {
```

Comment:
This wording change is interesting. The test doesn't define the action? The test is the result of an action. 

Opinion?

Reply:
I was trying to make it consistent with the other test names. Take a look at the console output when running tests:

```
  auro-pane
    ✔ is accessible
    ✔ defines the custom element
    ✔ parses 2020-09-09 correctly
    ✔ parses 2020-01-11 correctly
    ✔ parses invalid date correctly
```
Actually, looking at that, maybe we should rename the test to "is defined"?

I don't feel super strongly about it, so I can change it back if you want.

Reply:
I like where you are going. I look at test names like "is something happening ... ?" yes or no?

I am being nit-picky here as I really suck at tests. So, these are only opinions in the end.

Reply:
Since there don't seem to be any strong objections, I renamed the test to "is defined".